diff --git a/node_modules/@sveltejs/kit/src/exports/internal/event.js b/node_modules/@sveltejs/kit/src/exports/internal/event.js
index 9d65849..227e773 100644
--- a/node_modules/@sveltejs/kit/src/exports/internal/event.js
+++ b/node_modules/@sveltejs/kit/src/exports/internal/event.js
@@ -10,13 +10,13 @@ let sync_store = null;
 /** @type {AsyncLocalStorage<RequestStore | null> | null} */
 let als;
 
-import('node:async_hooks')
-	.then((hooks) => (als = new hooks.AsyncLocalStorage()))
-	.catch(() => {
-		// can't use AsyncLocalStorage, but can still call getRequestEvent synchronously.
-		// this isn't behind `supports` because it's basically just StackBlitz (i.e.
-		// in-browser usage) that doesn't support it AFAICT
-	});
+// import('node:async_hooks')
+// 	.then((hooks) => (als = new hooks.AsyncLocalStorage()))
+// 	.catch(() => {
+// 		// can't use AsyncLocalStorage, but can still call getRequestEvent synchronously.
+// 		// this isn't behind `supports` because it's basically just StackBlitz (i.e.
+// 		// in-browser usage) that doesn't support it AFAICT
+// 	});
 
 /**
  * Returns the current `RequestEvent`. Can be used inside server hooks, server `load` functions, actions, and endpoints (and functions called by them).
